docs/changelog.rst

```diff
@@ -186,6 +186,7 @@ New features:
 
 Fixes:
 
+* Small improvement to ``web`` plugin test coverage (for `expand` and `stats` features)
```

Comment:
We usually don't put testing stuff in the changelog because it's not user-facing (despite the reminder in the PR template 😳).

Reply:
OK. I have backed out the changelog. Let me know if it is now OK